website/docs/contributing/started.mdx

```diff
@@ -56,15 +57,27 @@ You can download it [here][code].
 </TabItem>
 <TabItem value="winget">
 
-The project has a [winget] [configuration][winget-configuration] file available to install the dependencies.
+The project has a [winget] [manifest][winget-manifest] file available to install the dependencies.
 Navigate to the cloned repository and use winget to install the dependencies:
 
 ```powershell
-winget configure configurations/configuration.dsc.yaml
+winget configure ./manifests/packages.winget.yml
 ```
 
 Done! Restart the terminal and you're ready to continue below.
 
+</TabItem>
+<TabItem value="scoop">
+
+The project has a [scoop] [manifest][scoop-manifest] file available to install the dependencies.
+Navigate to the cloned repository and use scoop to install the dependencies:
+
+```powershell
+scoop install ./manifests/packages.scoop.yml
```

Comment:
Scoop (I mean, the [original version][scoop]) doesn't support a YAML manifest, does it?

[scoop]: https://github.com/ScoopInstaller/Scoop